scalafix/input/src/main/scala/fix/v1_0_0/RemoveSplit.scala

```diff
@@ -0,0 +1,17 @@
+/*
+rewrite = "scala:fix.v1_0_0.RemoveSplit"
+ */
+package fix
+package to1_0_0
+
+import cats.implicits._
+import cats.arrow.Split
+import cats.syntax.split._
```

Comment:
The `split` syntax isn't actually used below, and if it was used it, this import would clash with `cats.implicits._`.

If you want to test the split syntax you would have

```scala
import cats.syntax.split._ // afterwards => import cats.syntax.arrow._
import cats.instances.function._

val f = toLong split toDouble
```